store/identity.ts

```diff
@@ -59,6 +60,13 @@ export const mutations = {
     // Set new subscription
     Vue.set(state, 'balanceSub', sub)
   },
+  changeAddressFormat(state: IdentityStruct, ss58Prefix: number): void {
+    if (state.auth.address) {
+      const address = formatAddress(state.auth.address, ss58Prefix)
+      Vue.set(state.auth, 'address', address)
```

Comment:
`this.$set` same with `Vue.set`, thus, no need to import `Vue`

Reply:
Not sure if `$set` is present in the vuex module

Reply:
you are right, wasn't notice it is vuex store.